daprdocs/content/en/getting-started/install-dapr-cli.md

```diff
@@ -48,6 +48,10 @@ Install the latest windows Dapr cli to `C:\dapr` and add this directory to the U
 ```powershell
 powershell -Command "iwr -useb https://raw.githubusercontent.com/dapr/cli/master/install/install.ps1 | iex"
 ```
+If the above command does not run try using: 
+```
+Set-ExecutionPolicy RemoteSigned -Scope CurrentUser -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/dapr/cli/master/install/install.ps1 '))
+```
```

Comment:
There are two different settings - ExecutionPolicy, and TLS configuration - that it's trying to set before running the main command. Not in favor of adding it here as (a) these settings can be environment specific and can be troubleshooted generally, and (b) "if the above command does not run" - can be due to a variety of reasons. A blanket solution should be avoided IMO.

However, if we were to go ahead with this, we should try giving specific error messages with specific solutions. The ExectionPolicy can also be scoped to `Process` instead of `CurrentUser`. For SecurityProtocol, `Tls12` can be used instead of `3072` (more readable). The final statement can be in the same form as line 49, i.e. using `iwr` and `iex` shorthands.

Reply:
Well understood on the environment specific observations. These were added generally due to the fact the  command did not work for me.
Could I implement your suggested changes and have it say, labelled as windows only ?

Reply:
@mukundansundar what are your thoughts on adding this?

Reply:
@duhowise Please add more details on what types of errors that this will solve or for which error logs should this workaround be used. 
@greenie-msft Should this be added to the Troubleshooting/Common Issues section instead of here?

Reply:
Will do